docs/docusaurus.config.js

```diff
@@ -12,7 +16,7 @@ const config = {
   favicon: "img/favicon.ico",
 
   // Set the production url of your site here
-  url: "https://code.groundlight.ai",
+  url: "https://www.groundlight.ai",
```

Comment:
Are we sure this is correct? I think this is the URL that will be the root of our project, including the rest of the docs website

Reply:
And thanks for catching this too

Reply:
This looks like maybe a merge mistake.  It should be `code`.